tests/cms/test_dummy.py

```diff
@@ -0,0 +1,2 @@
+def test_dummy(load_test_data):
+    pass
```

Comment:
```suggestion
def test_dummy(load_test_data):
    pass

```
I think newlines at end of files should always be present